Streetcode/Streetcode.BLL/Middleware/ApiRequestResponseMiddleware.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Streetcode.BLL.Interfaces.Logging;
+using Streetcode.BLL.Services.Logging;
+
+namespace Streetcode.BLL.Middleware
+{
+    public class ApiRequestResponseMiddleware : IMiddleware
+    {
+        private readonly ILoggerService _loggerService;
+        private readonly string[] _loggerPropertiesExceptions;
+
+        public ApiRequestResponseMiddleware(ILoggerService loggerService, IConfiguration configuration)
+        {
+            _loggerService = loggerService;
+            _loggerPropertiesExceptions = configuration.GetSection("Serilog:PropertiesToIgnore").Get<string[]>();
```

Comment:
Let's do on this way: If these properties are ignored by serilog - this section location for them is correct, if we have custom ignoring implementation - let's use our custom section in configuration